requirements.txt

```diff
@@ -22,8 +22,6 @@ cryptography==3.4.8
 coverage==6.4
 coverage-conditional-plugin==0.5.0
 httpx==1.0.0b0
-pytest-asyncio==0.15.1
```

Comment:
Should we add an explicit test dependency on `anyio`, since we now explicitly import it?

```
anyio==3.6.1
```